app/controllers/state_file/state_file_pages_controller.rb

```diff
@@ -8,7 +8,7 @@ def redirect_locale_home
     end
 
     def fake_direct_file_transfer_page
-      return render "public_pages/page_not_found", status: 404 if Rails.env.production? || Rails.env.staging?
+      return render "public_pages/page_not_found", status: 404 if Rails.env.production?
```

Comment:
Should this be `acts_like_production?`

Reply:
I thought so too at first but they will not have access to the IRS testing site so will have to go through fake-df flow more context here: https://cfastaff.slack.com/archives/C0544ERAFQV/p1706825778050309